services/blockchain-app-registry/shared/metadata.js

```diff
@@ -102,13 +101,36 @@ const getBlockchainAppsMetadata = async (params) => {
 		meta: {},
 	};
 
+	// Intialize db variables
+	params.whereIns = [];
```

Comment:
Please update `whereIns` everywhere within the code to `whereIn`.
```suggestion
	// Initialize DB variables
	params.whereIn = [];
```

Reply:
Applied.